homework/index.js

```diff
@@ -29,12 +31,115 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function addCont(login, img, contributeNumber, parent, link) {
+    const tr = createAndAppend('tr', parent);
+    const imageColumn = createAndAppend('th', tr);
+    createAndAppend('img', imageColumn, {
+      src: img,
+      class: 'contributorImg',
+    });
+    const name = createAndAppend('td', tr);
+    createAndAppend('a', name, {
+      href: link,
+      target: '_blank',
+      text: login,
+    });
+    const contributeNamePlace = createAndAppend('td', tr);
+    createAndAppend('span', contributeNamePlace, {
+      text: contributeNumber,
+      class: 'contributeNumber',
+    });
+  }
+
+  function addRow(property, key, parent) {
+    const tr = createAndAppend('tr', parent);
+    createAndAppend('th', tr, {
+      text: `${property} :`,
+    });
+    createAndAppend('td', tr, { text: key });
+    return tr;
+  }
+
+  let optionNumber = 0;
+  function madeOption(repo, selectButton) {
+    createAndAppend('option', selectButton, {
+      text: repo.name,
+      value: optionNumber,
+    });
+    optionNumber++;
+  }
+
+  function renderContributorDetails(conts, contributorsContainer) {
```

Comment:
Please do not use cryptic abbreviations such as `conts`. It only takes a few more characters to take away any doubt that you mean `constributors`.